docs/pricing/quotas/manage-seer-budget.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Manage Your Seer Budget
+sidebar_order: 40
+description: "Understand where and how to manage your Seer budget, see which projects are consuming more than others, and how to increase or decrease your spend."
+---
+
+<Alert>
+[Seer](/product/ai-in-sentry/seer/) is only available when purchased additionally while on a team or business plan. Once Seer has been purchased, your subscription will have $25 worth of Seer budget to be used across scans and Issue Fixes.
```

Comment:
"Once Seer has been purchased, your subscription will have $25 worth of Seer budget to be used across Issue Scans and Issue Fixes."